templates/deno/src/id.ts.twig

```diff
@@ -1,9 +1,30 @@
 export class ID {
+    // Generate an hex ID based on timestamp
+    // Recreated from https://www.php.net/manual/en/function.uniqid.php
+    static #hexTimestamp(): string {
+        const now = new Date();
```

Comment:
Seems to work, but some of the IDs aren't ordered, the timestamp bit is the same.